src/utils/stringifyVariables.ts

```diff
@@ -0,0 +1,46 @@
+const seen = new Set();
+
+const stringify = (x: any): string => {
+  if (x === undefined) {
+    return '';
```

Comment:
divergence from `fast-json-stable-stringify`: We don't expect to stringify `undefined`, so we can just return an empty string here, which makes the return type consistently a `string` and simplifies lines 19 and 34